CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 <!--
 # v0.2.0
 _2017_
-  - [Downloads for v0.2.0](https://github.com/okkur/dailymile/releases/tag/v0.1.0)
+  - [Downloads for v0.2.0](#downloads-for-v020)
```

Comment:
You are overwriting our manual changelog. Don't just git commit. Do a selective "git add -p" for the things we want.

Reply:
Still unfixed